### PR TITLE
docs(readme): comply new policy template INTER-295

### PR DESCRIPTION
**Pull request by Orkuncakilkaya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,11 +23,24 @@
 # Fingerprint Pro Cloudflare worker
 The Cloudflare worker is responsible for delivering the latest fingerprinting client-side logic and proxying identification requests and responses between your site and Fingerprint Pro's APIs.
 
+## Requirements
+
+This library needs these to be in order to build and run:
```
Comment (necipallef) [inline README.md]: We need to mention here that the integration backwards-compatibility is provided by Cloudflare, and there are only requirements for local development. We can mention [this field](https://github.com/fingerprintjs/fingerprintjs-pro-cloudflare-worker/blob/rc/wrangler.toml#L6) as well.

File: README.md [hunk 2]
```diff
@@ -23,11 +23,24 @@
...
+
+- Node 16 or Node 18
```
Comment (necipallef) [inline README.md]: I think we should mention Node20 and can even remove Node 16

File: README.md [hunk 3]
```diff
@@ -23,11 +23,24 @@
...
+- [Wrangler](https://developers.cloudflare.com/workers/wrangler/install-and-update/)
+
 ## Setup process
 The process consists of two steps. One needs to set up worker creation together with our support at [support@fingerprint.com](mailto:support@fingerprint.com). Afterwards, the Fingerprint Pro JS agent on the site needs to be configured to communicate with the worker.
 
 You can find the full documentation at [https://dev.fingerprint.com/docs/cloudflare-integration](https://dev.fingerprint.com/docs/cloudflare-integration).
 
+## Support
+
+To report problems, ask questions or provide feedback, please use [Issues](https://github.com/fingerprintjs/fingerprintjs-pro-cloudflare-worker/issues). If you need private support, you can email us at [oss-support@fingerprint.com](mailto:oss-support@fingerprint.com).
```
Comment (necipallef) [inline README.md:50]: Is `oss-support@fingerprint.com` the correct email address here?

File: README.md [hunk 4]
```diff
@@ -23,11 +23,24 @@
...
 The Cloudflare worker is responsible for delivering the latest fingerprinting client-side logic and proxying identification requests and responses between your site and Fingerprint Pro's APIs.
 
+## Requirements
+
+This library needs these to be in order to build and run:
+
+- Node 16 or Node 18
+- Typescript 4
+- Playwright (with a headless browser preferably chromium)
+- [Wrangler](https://developers.cloudflare.com/workers/wrangler/install-and-update/)
```
Comment (necipallef) [inline README.md]: There are different `wrangler` versions out there, `v1`, `v2` and `v3`. In my case, I started using `v1` and then switched to `v2`, and never tried `v3`. We probably support `v2` and `v3` and not `v1` but need to make sure and mention it here.

**Review by necipallef — CHANGES_REQUESTED** [review]
left comments

File: README.md (hunk 3 above)
Comment (Orkuncakilkaya) [reply README.md:50]: Yes, according the guideline "Writing DX GitHub Readmes" provided by Juraj at our Notion documents.

File: README.md (hunk 4 above)
Comment (Orkuncakilkaya) [reply README.md]: We support v3+ I mentioned in the readme

File: README.md (hunk 2 above)
Comment (Orkuncakilkaya) [reply README.md]: Done 👍

File: README.md (hunk 1 above)
Comment (Orkuncakilkaya) [reply README.md]: Updated with even better, as Juraj Suggested I extended requirements section with two subsections called "Usage Requirements" and "Development Requirements".

**Review by necipallef — DISMISSED** [review]

File: README.md [hunk 5]
```diff
@@ -23,11 +23,30 @@
...
+### Development Requirements
+
+This library needs these to be in order to build and run:
+
+- Node 20
+- Typescript 4+
+- Playwright
+- [Wrangler](https://developers.cloudflare.com/workers/wrangler/install-and-update/) v3+
+
 ## Setup process
```
Comment (JuroUhlar) [inline README.md]: * This seems outdated, you don't need to contact support to setup the Cloudflare integration. 
* The Readme should point people to https://dev.fingerprint.com/docs/cloudflare-integration follow the set up guide using the installation wizard.

File: README.md [hunk 6]
```diff
@@ -23,11 +23,30 @@
 # Fingerprint Pro Cloudflare worker
 The Cloudflare worker is responsible for delivering the latest fingerprinting client-side logic and proxying identification requests and responses between your site and Fingerprint Pro's APIs.
 
+## Requirements
+
+### Usage Requirements
+
+Please see the [Setup Process](#setup-process) section
```
Comment (JuroUhlar) [inline README.md]: The integration actually has a well defined usage requirements defined https://dev.fingerprint.com/docs/cloudflare-integration#setup 

You could briefly list them here and point to the guide for more details

File: README.md [hunk 7]
```diff
@@ -23,11 +23,30 @@
...
+
+### Development Requirements
```
Comment (JuroUhlar) [inline README.md]: The information hierarchy is off here, Setup/Getting started is universal and much more important than Development requirements. Consider moving them below or to `contributing.md`

**Review by JuroUhlar — CHANGES_REQUESTED** [review]

File: README.md [hunk 8]
```diff
@@ -23,11 +23,30 @@
 # Fingerprint Pro Cloudflare worker
 The Cloudflare worker is responsible for delivering the latest fingerprinting client-side logic and proxying identification requests and responses between your site and Fingerprint Pro's APIs.
```
Comment (JuroUhlar) [inline README.md]: Consider aligning the introduction and setup sections with Cloudfront: https://github.com/fingerprintjs/fingerprint-pro-cloudfront-integration 
and Azure https://github.com/fingerprintjs/fingerprint-pro-azure-integration

Comment (Orkuncakilkaya) [reply README.md]: Updated description

File: README.md (hunk 7 above)
Comment (Orkuncakilkaya) [reply README.md]: Moved development requirements inside contributing.md file

File: README.md (hunk 6 above)
Comment (Orkuncakilkaya) [reply README.md]: Updated usage requirements

File: README.md (hunk 5 above)
Comment (Orkuncakilkaya) [reply README.md]: Updated here

File: README.md [hunk 9]
```diff
@@ -21,13 +21,36 @@
...
-The Cloudflare worker is responsible for delivering the latest fingerprinting client-side logic and proxying identification requests and responses between your site and Fingerprint Pro's APIs.
+
+[Fingerprint](https://fingerprint.com/) is a device intelligence platform offering 99.5% accurate visitor identification.
+
+Fingerprint Pro Cloudflare Integration is responsible for
+
+- Proxying download requests of the latest Fingerprint Pro JS Agent between your site and Fingerprint CDN.
+- Proxying identification requests and responses between your site and Fingerprint Pro's APIs.
+
+This [improves](https://dev.fingerprint.com/docs/cloudfront-proxy-integration#the-benefits-of-using-the-cloudfront-integration) both accuracy and reliability of visitor identification and bot detection on your site.
```
Comment (JuroUhlar) [inline README.md]:
```suggestion
This [improves](https://dev.fingerprint.com/docs/cloudflare-integration#the-benefits-of-using-the-cloudflare-integration) both accuracy and reliability of visitor identification and bot detection on your site.
```

**Review by ilfa — DISMISSED** [review]

File: README.md [hunk 10]
```diff
@@ -21,13 +21,36 @@
...
+## Requirements
+
+### Usage Requirements
+
+- Credentials from Cloudflare
+  - [Cloudflare Account ID](https://dev.fingerprint.com/docs/cloudflare-integration#cloudflare-account-id)
+  - [Cloudflare API Token](https://dev.fingerprint.com/docs/cloudflare-integration#cloudflare-api-token)
+- FingerprintPro Account
+
+By just following the wizard with your credentials it's easy to setup. Please see the [Setup Process](#setup-process) section
```
Comment (JuroUhlar) [inline README.md]: @Orkuncakilkaya These are not the requirements listed in the docs. Getting Cloudflare account ID and token is covered in the guide, therefore these are not requirements. Requirements or pre-requisites are "stuff the guide/tool assumes you already have before you start", not "stuff the guide helps you set up".

```suggestion
* [Fingerprint Pro account](https://dashboard.fingerprint.com/signup) with the _Owner_ role assigned.
* A website served by Cloudflare. For maximum accuracy benefits, your website should be [proxied by Cloudflare](https://developers.cloudflare.com/dns/manage-dns-records/reference/proxied-dns-records/) (not DNS-only). 

See the [Cloudflare integration guide](https://dev.fingerprint.com/docs/cloudflare-integration#setup) for more details. 

```

File: README.md [hunk 11]
```diff
@@ -21,13 +21,36 @@
...
 
 ## Setup process
```
Comment (JuroUhlar) [inline README.md]:
```suggestion
## How to install
```

File: README.md [hunk 12]
```diff
@@ -21,13 +21,36 @@
...
-The process consists of two steps. One needs to set up worker creation together with our support at [support@fingerprint.com](mailto:support@fingerprint.com). Afterwards, the Fingerprint Pro JS agent on the site needs to be configured to communicate with the worker.
 
+Setup process is simple enough. You just follow a setup wizard and it's ready to use.
 You can find the full documentation at [https://dev.fingerprint.com/docs/cloudflare-integration](https://dev.fingerprint.com/docs/cloudflare-integration).
```
Comment (JuroUhlar) [inline README.md]: Tech writing tips: 
* Don't make qualitative statements like "this is simple", that's highly subjective
* Links should have meaningful text, not URLs 


```suggestion
You can install the Cloudflare integration using an [installation wizard](https://dashboard.fingerprint.com/integrations) in the Fingerprint dashboard. 

See the [Cloudflare integration guide](https://dev.fingerprint.com/docs/cloudflare-integration#setup) for more details.
```

**Review by JuroUhlar — CHANGES_REQUESTED** [review]

**Review by JuroUhlar — APPROVED** [review]

**Review by sshelomentsev — APPROVED** [review]

**Pull request merged**